db/seeds.rb

```diff
@@ -1,31 +1,25 @@
 @service ||= PropublicaService.new
```

Comment:
@jmejia My seeds file is getting to be pretty involved. Is there a better way I should be handling this? Does it make sense to wrap some of these methods in a class even if they're only being used to seed?